CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+- [#1980](https://github.com/FuelLabs/fuel-core/pull/1980): Add `Transaction` to relayer 's event filter
```

Comment:
No, the current contract deployments do not support FTI.

I am currently developing it and the integration tests fail because this code is missing.

I thought FTI was already supported by fuel-core? In fact, after doing these changes, I have been able to trigger transactions through the relayer, locally.

Reply:
Yeah, we thought that we implemented everything=D But it seems we forgot to update the request to Ethereum RPC)))

Okay, if any network has no support for FTI, then we don't need any additional code.